### PR TITLE
PRL-4467_Events_Court_Admin_submittedt

### PR DESCRIPTION
**Pull request by JamesCutts00**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-4467

### Change description ###

Changes to events for court admin when the case is in the submitted state.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/private-law/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -856,7 +856,7 @@
     "Name": "Add case number",
     "Description": "CTSC staff add family man number for the case.",
     "DisplayOrder": 3,
-    "PreConditionState(s)": "SUBMITTED_PAID;AWAITING_RESUBMISSION_TO_HMCTS;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
+    "PreConditionState(s)": "AWAITING_RESUBMISSION_TO_HMCTS;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```
Comment (yogendraU) [inline definitions/private-law/json/CaseEvent/CaseEvent.json:859]: do we need AWAITING_RESUBMISSION_TO_HMCTS state?

File: definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-draft-order-dropdown",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/submitted",
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_RESUBMISSION_TO_HMCTS;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```
Comment (yogendraU) [inline definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json:45]: do we need AWAITING_RESUBMISSION_TO_HMCTS state?

File: definitions/private-law/json/CaseEvent/Order/CaseEvent-prod.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-draft-order-dropdown",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/submitted",
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_RESUBMISSION_TO_HMCTS;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```
Comment (yogendraU) [inline definitions/private-law/json/CaseEvent/Order/CaseEvent-prod.json:45]: do we need AWAITING_RESUBMISSION_TO_HMCTS state?

**Comment by yogendraU** [comment]
hi @JamesCutts00 , please confirm my comments with @shashisk97

**Review by yogendraU — CHANGES_REQUESTED** [review]

**Review by yogendraU — APPROVED** [review]

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**